tests/test_content_loader.py

```diff
@@ -423,55 +499,25 @@ def test_get_question_by_slug(self):
 
         assert content.get_question_by_slug('section-two-question').get('id') == 'q2'
 
-    def test_get_next_section(self):
-        content = ContentManifest([
-            {
-                "slug": "first_section",
-                "name": "First section",
-                "questions": [{
-                    "id": "q1",
-                    "question": 'First question',
-                    "depends": [{
-                        "on": "lot",
-                        "being": ["SCS", "SaaS", "PaaS"]
-                    }]
-                }]
-            },
-            {
-                "slug": "second_section",
-                "name": "Second section",
-                "questions": [{
-                    "id": "q2",
-                    "question": 'First question',
-                    "depends": [{
-                        "on": "lot",
-                        "being": ["SCS", "SaaS", "PaaS"]
-                    }]
-                }]
-            },
-            {
-                "slug": "third_section",
-                "name": "Third section",
-                "editable": True,
-                "questions": [{
-                    "id": "q3",
-                    "question": 'First question',
-                    "depends": [{
-                        "on": "lot",
-                        "being": ["SCS", "SaaS", "PaaS"]
-                    }]
-                }]
-            },
-        ])
-
-        assert content.get_next_section_id() == "first_section"
-        assert content.get_next_section_id("first_section") == "second_section"
-        assert content.get_next_section_id("second_section") == "third_section"
-        assert content.get_next_section_id("third_section") is None
-
-        assert content.get_next_editable_section_id() == "third_section"
-        assert content.get_next_editable_section_id(
-            "second_section") == "third_section"
+    def test_get_next_section(self, manifest_with_sections):
+        assert manifest_with_sections.get_next_section_id() == "first_section"
+        assert manifest_with_sections.get_next_section_id("first_section") == "second_section"
+        assert manifest_with_sections.get_next_section_id("second_section") == "third_section"
+        assert manifest_with_sections.get_next_section_id("third_section") == "fourth_section"
+        assert manifest_with_sections.get_next_section_id("fourth_section") == "fifth_section"
+        assert manifest_with_sections.get_next_section_id("fifth_section") is None
+
+    def test_get_next_editable_section(self, manifest_with_sections):
+        assert manifest_with_sections.get_next_editable_section_id() == "third_section"
+        assert manifest_with_sections.get_next_editable_section_id("second_section") == "third_section"
```

Comment:
Might there be value in adding `assert manifest_with_sections.get_next_editable_section_id("first_section") == "third_section"`?